configure.ac

```diff
@@ -70,13 +70,14 @@ OPAL_LOAD_PLATFORM
 # Start it up
 #
 
+OPAL_CONFIGURE_SETUP
```

Comment:
I noticed that this was actually in the wrong order while working on this (i.e., Perl was checked before the subtitle instead of the other way around).

I can separate it out into a separate commit, if you prefer.

Reply:
no, fair enough.

Reply:
Yeah, but now I can't un-see it.  Fix inbound.